### ISSUE TITLE
Drupal 7.59 Moodle 3.5 Only showing website URL

### ISSUE THREAD

**Issue by choren5150** [cannod/moodle-drupalservices#83]
I know there was another thread on this, but it didn't really have an answer. I have been toying with this for a week trying to get this module working with Drupal 7.59 and Moodle 3.5, but it only ever shows the Drupal Website URL and no status or mapping, etc. I have looked through the issues page and tried a few things there, tried different branches of the module, and installed an older version of Drupal and Moodle to see what I should see when it is functioning properly. Unfortunately, as I work for a medical school I need to remain HIPPA compliant so i cannot drop to a very old version of Moodle to make this work.

I have gone through the guide multiple times and triple checked the configs, but it will just not show any information beyond the URL. I have installed it through both CLI and CMS just to see if the effects would be different and also manually entered the information into the code to see if I could get it to kick start the module. Any ideas on where to go or any logs I can pull for you guys to see if it can give you any insight?

**Comment by monishdeb** [comment]
I am encountering the same issue :(

**Comment by choren5150** [comment]
@monishdeb any luck on this on your end? Kind of stuck and haven't had much time to push more on it.

**Comment by monishdeb** [comment]
@choren5150 yes I made it work by pulling the PR https://github.com/cannod/moodle-drupalservices/pull/78


Please check my comments at https://github.com/cannod/moodle-drupalservices/issues/77
